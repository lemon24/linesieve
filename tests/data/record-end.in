--record-start { --record-end }
sub '^(.*)' '[\1]'
---
zero
{ one
two
} three
four
five
{
six
}
seven
