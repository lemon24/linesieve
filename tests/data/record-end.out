[zero]
[{ one
two
} three]
[four
five]
[{
six
}]
[seven]
